example-apps/my-nest-project/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.8"
```

Comment:
what about creating a second example-app with db? that way we also have an example without db, which makes life easier for consuming

Reply:
makes sense